docs/application/dotnet/guides/multimedia/audio.md

```diff
@@ -140,8 +140,34 @@ To query audio device information:
         > **Note**   
 		> The initial running state of the connected device is `false`, which means the connected device is not running.
 
+-   More functionality for AudioDeviceType.UsbAudio and AudioDeviceIoDirection.Output device, you can utilize more functionality with the following methods of the [Tizen.Multimedia.AudioDevice](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.AudioDevice.html) class:
```

Comment:
Suggested rephrase:

To utilize additional functionality for `AudioDeviceType.UsbAudio` and `AudioDeviceIoDirection.Output`, use the following methods of the [Tizen.Multimedia.AudioDevice](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.AudioDevice.html) class:

Kindly verify the technical meaning before incorporating suggested rephrase.

Reply:
The useOutputDevice is instance of AudioDevice. I'll add code snippet to get the device of UsbAudio and Output direction to avoid misunderstanding.

Reply:
Another question. 
So, is us**b**OutputDevice the misstype of useOutputDevice? 
In the code snippet and description I can see Us**b**Audio and us**b**OutputDevice.
Maybe it should be "use", not "usb"?

Reply:
It's typo in my comment. usbOutputDevice is right as well as code snippet.